src/main.jsx

```
@@ -1,16 +1,15 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.jsx";
+import { createRoot } from "react-dom/client";
 import "./index.css";
 import { ThemeProvider } from "./context/ThemeContext.jsx";
 import { Toaster } from "./components/ui/sonner.tsx";
+import { ThemeProvider } from "./components/ThemeProvider.jsx";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ThemeProvider'. Consider to delete it or rename it.

'ThemeProvider' is defined here:


(lint/suspicious/noRedeclare)